### DISCUSSION TITLE
Issue with PostgreSQL Connection in Lowdefy When Hosted on Cloud Services"

### DISCUSSION THREAD

**Discussion by ClorisYT** [lowdefy/lowdefy#1848] [Questions and Help]
Hello！

I've encountered an issue where my PostgreSQL connection in Lowdefy fails to execute properly if PostgreSQL is hosted on a cloud service (like Docker) rather than localhost. Is this a limitation of Lowdefy?

**Comment by Gervwyk** [comment]
No, there is no such limitation. I suspect something else is causing this. Please share more info?

**Reply by ClorisYT** [reply]
I mean I just change the LOWDEFY_SECRET_PG_CONNECTION_STRING="postgresql://postgres:root@localhost:5432/test_db"
to "postgresql://postgres:xxxx@10.136.218.632:5432/test_db" (the vm url)

and when I run pnpm build:app, it will show no Build_ID
